openapi/src/parts/azure_landing_zone.yaml

```diff
@@ -0,0 +1,196 @@
+# Azure Landing Zones paths
+paths:
+  /api/azure/landingzones/v1:

```

Comment:
Maybe it should be:
```
/api/landingzones/v1/azure
```
?

* landingzone is the resource
* it's versioned at the resource level
* azure in the path looks forward to when we might have landing zones on more than 1 cloud.
   * could also not have azure in the path and put `cloudPlatform` in the request body, similar to [createCloudContext](https://workspace.dsde-dev.broadinstitute.org/swagger-ui.html#/Workspace/createCloudContext)

Reply:
My assumption was that we might have landing zones for different cloud providers. Also, even if that's not the plan "azure" prefix adds additional context, otherwise it's not clear where a landing zone would reside. Probably just "azurelandingzones" makes sense if we don't have plan for other cloud providers.

Reply:
Yes, we may have landing zones for different cloud providers. When we took the LZ direction, we were not assuming that an LZ for GCP would look anything like an LZ for Azure. This interface is very generic, so I am biased to agree with @rtitle that the cloud could be an attribute of the LZ.

Reply:
@giventocode what is your opinion about this?

Reply:
I agree that we don't have any planned or even tentative functionality that benefits from having `azure` at the root of the URI. I like Rob's suggestion of `/api/landingzones/v1/azure`.

Reply:
I vote for /api/landingzones/v1/azure - it states clear the concept<landingzones> and supported cloud for a version <v1/azure>. While <azurelandingzone> is a mix of implementation/cloud availability and a concept. If landing zone stays as a pure Azure concept, both variants are equal, however, if the concept will get support in other clouds, the  /api/landingzones/version/cloud is logical.

Reply:
Thanks for the feedback. I vote to Rob's suggestion too. So, it will be `/api/landingzones/v1/azure`